CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+
+#### New features and UX changes
+
+- stage1: implement read-only rootfs ([#2624](https://github.com/coreos/rkt/pull/2624)). Using the Pod manifest readOnlyRootFS option mounts the rootfs of the app as read-only using systemd-exec unit option ReadOnlyDirectories.
+- stage1: capabilities: implement both remain set and remove set ([#2589](https://github.com/coreos/rkt/pull/2589)). It follows the [Linux Isolators semantics from the App Container Executor spec](https://github.com/appc/spec/blob/master/spec/ace.md#linux-isolators), as modified by https://github.com/appc/spec/pull/600.
+- stage1/init: create a new mount ns for each app ([#2603](https://github.com/coreos/rkt/pull/2603)). Up to this point, you could escape the app's chroot easily by using a simple program downloaded from the internet [1](http://www.unixwiz.net/techtips/chroot-practices.html). To avoid this, we now create a new mount namespace per each app.
+- api: Return the pods even when we failed getting information about them ([#2593](https://github.com/coreos/rkt/pull/2593)).
+- stage1/usr_from_coreos: use CoreOS 1032.0.0 with systemd v229 ([#2514](https://github.com/coreos/rkt/pull/2514)).
+
+#### Bug fixes
+
+- kvm: fix flannel network info ([#2625](https://github.com/coreos/rkt/pull/2625)). It wasn't saving the network information on disk.
+- Machine name wasn't being populated with the full UUID ([#2575](https://github.com/coreos/rkt/pull/2575)).
+- Some simple arg doc string fixes ([#2588](https://github.com/coreos/rkt/pull/2588)). Remove some unnecessary indefinite articles from the start of argument doc strings and fixes the arg doc string for run-prepared's --interactive flag.
+- stage1: Fix segfault in enterexec ([#2608](https://github.com/coreos/rkt/pull/2608)). Prior to this commit, if `rkt enter` was executed without the `TERM` environment variable set, a segfault could occur. This commit treats an unset TERM environment variable as meaning 'xterm' and avoids the segfault. This happened if rkt enter was executed without the TERM environment variable set.
+- net: fix port forwarding behavior with custom CNI ipMasq'ed networks different and allow different hostPort:podPort combinations ([#2387](https://github.com/coreos/rkt/pull/2387)). This fixes port forwarding functionality when a pod is run with custom ipMasq'ed CNI networks. It also fixes #2283: port forwarding where hostPort and podPort are different.
+- stage0: check and create /etc ([#2599](https://github.com/coreos/rkt/pull/2599)). Checks '/etc' before writing to '/etc/rkt-resolv.conf' and creates it with default permissions if it doesn't exist.
+
+#### Other changes
+
+- godep: update cni to v0.2.3 ([#2618](https://github.com/coreos/rkt/pull/2618)).
+- godep: update appc/spec to v0.8.1 ([#2623](https://github.com/coreos/rkt/pull/2623), [#2611](https://github.com/coreos/rkt/pull/2611)). This also fixes some compilation errors introduced by the following changes:
```

Comment:
The part after "This also fixes" is not really relevant for the changelog, remove it plz